proto/greenfield/virtualgroup/types.proto

```diff
@@ -73,6 +73,13 @@ message GVGStatisticsWithinSP {
   uint32 break_redundancy_reqmt_gvg_count = 4;
 }
 
+message GVGFamilyStatisticsWithinSP {
+  // storage_provider_id defines the id of the sp which the statistics associated to
+  uint32 storage_provider_id = 1;
```

Comment:
the `storage_provider_id` is not needed as the key is the sp_id already.

Reply:
fixed